apps/passport-client/test/stateValidation.spec.ts

```diff
@@ -1,14 +1,17 @@
+// todo:
+// - check that everywhere a v3 identity is present, a v4 identity should also be present
+// - check thatv4 identity is derived from v3 identity
+
```

Comment:
I suggest doing a search that each of the new invalid state error strings in hte checker appears somewhere in this file in a test.